src/cursor/cur_table.c

```diff
@@ -810,9 +813,10 @@ __wt_table_range_truncate(WT_CURSOR_TABLE *start, WT_CURSOR_TABLE *stop)
     }
 
     /* Truncate the column groups. */
-    for (i = 0; i < WT_COLGROUPS(ctable->table); i++)
+    for (i = 0; i < WT_COLGROUPS(ctable->table); i++) {
```

Comment:
This change is not necessary. Were the braces added for debugging statements and then left in?